tests/scss/base/_base-theme.scss

```diff
@@ -0,0 +1,11 @@
+/*  Base Theme  */
```

Comment:
we should have something similar for each language supported.  Would be
helpful getting a start on js  to see an equivalent js structure.

On Wed, Mar 11, 2015 at 7:48 PM, Joel Kemp <notifications@github.com> wrote:

> In tests/scss/base/_base-theme.scss
> <https://github.com/mrjoelkemp/Dependents/pull/131#discussion_r26274807>:
>
> > @@ -0,0 +1,11 @@
> > +/*  Base Theme  */
>
> This is all pretty great. I wanted this for a bit but opened external
> projects for the longest time. I could even potentially write some
> automated tests using these fixtures.
>
> Many thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mrjoelkemp/Dependents/pull/131/files#r26274807>.
>



-- 
Cheers, Cuidate, TTFN <http://www.youtube.com/watch?v=5Gu50vq5ux4>
David